docs/db_dev_process/overview.rst

```diff
@@ -25,11 +25,30 @@ By looking at all the resulting specimens, you can correlate which bits in which
 
 Looking at the implemented design in Vivado with "Show Routing Resources" turned on is quite helpful in understanding what all choices exist.
 
-SymbiFlow/prjxray/tools/
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. toctree::
+   :maxdepth: 2
```

Comment:
Change to maxdepth: 1. I feel this is a bit too visually noisy. I really only expect a list of the fuzzers, not the section headers as well.

Reply:
Fixed.